### PR TITLE
Tests for CollectOxoGMetrics, CollectJumpingLibraryMetrics and BamTBfq classes

### PR DESCRIPTION
**Pull request by ekazachkova**
Current version Picard Tools doesn't contain test suite for CollectOxoGMetrics, CollectJumpingLibraryMetrics and BamToBfq classes. We represent PR that contains acceptance tests for this classes and new test data for BamToBfq class.

### CODE REVIEW

File: src/test/java/picard/analysis/CollectOxoGMetricsTest.java [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+                "REFERENCE_SEQUENCE=" + REFERENCE_SEQUENCE.getAbsolutePath()
+        };
+        CollectOxoGMetrics collectOxoGMetrics = new CollectOxoGMetrics();
+        Assert.assertEquals(collectOxoGMetrics.instanceMain(args), 0,
+                "Can't process " + SAM_FILE.getAbsolutePath() + " correctly");
+
+        final MetricsFile<CollectOxoGMetrics.CpcgMetrics, Comparable<?>> output = new MetricsFile<>();
+        output.read(new FileReader(outputFile));
+
+        final CollectOxoGMetrics.CpcgMetrics cpcgMetrics = output.getMetrics().get(4);
```
Comment (fleharty) [inline src/test/java/picard/analysis/CollectOxoGMetricsTest.java]: It's not clear what index 4 refers to, could you create a variable with a useful name?

**Comment by ekazachkova** [comment]
Done. @fleharty have a look, please.

**Comment by fleharty** [comment]
@ekazachkova Feel free to rebase, squash, and merge!

**Comment by yfarjoun** [comment]
@fleharty you should merge if you are sure it's fine.

**Comment by ekazachkova** [comment]
Thanks! @fleharty you can merge.

**Pull request merged**